src/Model/Item.php

```diff
@@ -1445,7 +1445,24 @@ public static function isValid(array $item)
 			Logger::notice('Causer is ignored by user', ['causer-link' => $item['causer-link'], 'uid' => $item['uid'], 'item-uri' => $item['uri']]);
 			return false;
 		}
-		
+
+		$toplevelOwnerId = null;
+		if ($item['parent-uri'] != $item['uri']) {
```

Comment:
Please use the `gravity` to check for the parent. I want to unify this check that way.

Reply:
Ok, I'll update the other instances in the same method.